python/dgl/distributed/partition.py

```diff
@@ -1410,8 +1554,21 @@ def init_type_per_edge(graph, gpb):
     return etype_ids
 
 
+def _load_parts(part_config, part_id, parts):
+    """load parts from variable or dist."""
+    if parts is None:
+        graph, _, _, _, _, _, _ = load_partition(
+            part_config, part_id, load_feats=False
+        )
+    else:
+        graph = parts[part_id]
+    return graph
+
+
 def gb_convert_single_dgl_partition(
     part_id,
+    parts,
```

Comment:
for new parameters, we usually add to the end with default values. and you need to update the docstring accordingly.